sdk/frontdoor/Azure.ResourceManager.FrontDoor/src/Generated/Models/VariableName.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.FrontDoor.Models
+{
+    /// <summary> Describes the supported variable for group by. </summary>
+    public readonly partial struct VariableName : IEquatable<VariableName>
```

Comment:
The name is too general, please rename it to be more specific to avoid conflicts with other RPs, such as prepend PR name. You can refer to `prepend-rp-prefix` in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md